.ci/jenkins/prepare_environment.sh

```diff
@@ -16,13 +16,19 @@
 # otherwise. Any license under such intellectual property rights must be express
 # and approved by Intel in writing.
 
+set -x
+
 # Install nGraph in /root/ngraph
 cd /home
 if [ -e ./ngraph ]; then
     cd ./ngraph
     # If ngraph repo is up to date, and wheel exist - no need to rebuild it so exit
     if [[ $(git pull) == *"Already up-to-date"* && -n $(find /home/ngraph/python/dist/ -name 'ngraph*.whl') ]]; then
         exit 0
+    else
+    # Remove old wheel
+    NGRAPH_WHL = $(find /home/ngraph/python/dist/ -name 'ngraph*.whl' -printf '%Ts\t%p\n' | sort -nr | cut -f2)
```

Comment:
Similarly here. Try `find /home/ngraph/python/dist/ -name 'ngraph*.whl' -print | sort -n | head -n1`

Reply:
It's supposed to delete all older files so I suppose I should simplify it to
`rm /home/ngraph/python/dist/ngraph*.whl'